woocommerce/woocommerce
# Update WP version check in product browse test

### Changes proposed in this Pull Request:

This PR updates the WP version check as before the check would fail and the test would fail to run--regardless of the WordPress version.

### How to test the changes in this Pull Request:

1. Verify the CI runs and passes
2. Before switching to this branch, set up a test site with either WordPress 5.8 or 5.8.1 and run the test, noting it is skipped:

```
npx wc-e2e test:e2e ./tests/e2e/specs/front-end/product-browse-search-sort.test.js

Test Suites: 1 skipped, 0 of 1 total
Tests:       3 skipped, 3 total
Snapshots:   0 total
Time:        3.229s
```

3. Set up a test site with WordPress 5.7.x and run the test, verify it is skipped as expected for a version less than 5.8:

```
npx wc-e2e test:e2e ./tests/e2e/specs/front-end/product-browse-search-sort.test.js
```

4. Update to WordPress 5.8 and verify the test runs:

```
npx wc-e2e test:e2e ./tests/e2e/specs/front-end/product-browse-search-sort.test.js
```

5. Update to WordPress 5.8.1 and verify the test runs:

```
npx wc-e2e test:e2e ./tests/e2e/specs/front-end/product-browse-search-sort.test.js
```
